container-templates/dockerfile/.devcontainer/devcontainer.json

```diff
@@ -9,9 +9,7 @@
 	},
 
 	// Set *default* container specific settings.json values on container create.
-	"settings": { 
-		"terminal.integrated.shell.linux": null
```

Comment:
What was this meant to accomplish? Overriding the user setting in case it's not installed?

Reply:
Correct. The fallback eliminates the need for it.